docs/local_development.md

```diff
@@ -100,13 +100,16 @@ yarn run dev:chrome # or dev:firefox, dev:opera
 
 - pull latest from `main` and run the setup script from the root directory `./setup.ts`.
 - Restart the node and redeploy contracts
+- Delete `aggregator.sqlite` in `./aggregator`. This is the local DB which will get regenerated when the aggregator is started.
 - Restart the aggregator and add the "-r" flag to the command e.g `./programs/aggregator.ts -r`.
 - Reset the Quill extension in your browser if you're developing with Quill. You can do this by removing the extension and then re-adding via "Load unpacked" again. Or run `debug.reset();` twice in the background page console.
 
 ### Additional troubleshooting tips
 
 - In general, the bundle or submission issues we've encountered have been us misconfiguring the data in the bundle or not configuring the aggregator properly.
 - Be careful using Hardhat accounts 0 and 1 in your code when running a local aggregator. This is because the local aggregator config uses the same key pairs as Hardhat accounts 0 and 1 by default. You can get around this by not using accounts 0 and 1 elsewhere, or changing the default accounts that the aggregator uses locally.
+- When packages are updated in the aggregator, you'll need to reload the deno cache as the setup script won't do this for you. You can do this with `deno cache -r deps.ts` in the `./aggregator` directory.
```

Comment:
Nah it's not quite right. The cache logic only says it's ok to use a previous fetch of the same url. If the url is different, it'll refetch it. No cache clearing needed.

The problem is that a lot of urls in the dependency tree have semver-matching urls (e.g. `@^1.0.0`) instead of being pinned (e.g. `@1.0.0`). This can lead to some dependencies being more up-to-date than others, causing conflicts. There's also other ways urls can resolve to different things over time, like updates to esm.sh itself.

I wouldn't go into detail in the docs, instead just something like:

```
- Sometimes there are issues related to the deno cache. You can clear it with `deno cache -r deps.ts test/deps.ts` in the `./aggregator` directory.
```

Reply:
Forgot to reply to this. I see, that makes sense, thanks for pointing this out. Will update in the current work I'm doing